### DISCUSSION TITLE
Wireguard on Debian 12 bookworm

### DISCUSSION THREAD

**Discussion by kevin-snt** [tteck/Proxmox#3263] [Help Needed] [answered]
Hi, I tried to install wireguard using the helper script while choosing debian 12, but it failed to install.
It didn't find wireguard-dkms

I think it works with debian 11 (not testing yet but it's the default value), but debian 12 was released a year ago and it seems to be more futur-proof to use it.

Is there any chance that this script supports it ?
Thanks

**Comment by tteck** [comment, marked as answer]
The script supports what the application requires.
